readme.md

```diff
@@ -20,4 +20,15 @@ This repository is licensed under the Apache 2.0 License, except for the [server
 
 ## Development
 
+This plugin contains both a server and web app portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) for more information about developing and extending plugins.
+
+For more information about contributing to Mattermost, and the different ways you can contribute, see https://www.mattermost.org/contribute-to-mattermost.
+
+## Support
+
+- For Mattermost customers - Please open a support case.
+- For questions, suggestions, and help, visit the [Jira Plugin channel](https://community.mattermost.com/core/channels/jira-plugin) on our Community server.
+- To report a bug, please [open an issue](https://github.com/mattermost/mattermost-plugin-jira/issues).
+
 Please read the Jira plugin's [developer docs](https://mattermost.gitbook.io/plugin-jira/development/environment) to learn more about contributing to the project.
```

Comment:
That doesn't seem right. Shouldn't that text replace the first part in `Development`?